en/privacy/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: "Privacy Policy for ruby-lang.org"
+lang: en
+---
+
+This privacy policy covers ruby-lang.org.
+
+## Email
+
+We will not give away your email address to anyone, who is not related to the operations of ruby-lang.org. We will also never ask you to send us any of your passwords via email.
+
+## Logfiles
+
+ruby-lang.org record access logs of the requests that reach the web servers, but we use those files for issue investigation and statistical purposes.
+
+## Cookies
+
+ruby-lang.org uses cookies to keep track of user preferences. Unless you login on the site, the cookies will not be used to store personal information and we do not give away the information from the cookies.
```

Comment:
> ruby-lang.org uses cookies to keep track of user preferences.

Do we? I don't think w.r-l.o has a user preference like https://www.php.net/my.php.

```suggestion
Some sites under ruby-lang.org (e.g. bugs.ruby-lang.org) uses cookies to keep track of user preferences. Unless you login on the site, the cookies will not be used to store personal information and we do not give away the information from the cookies.
```

Reply:
Yes, I noticed this for bugs.ruby-lang.org, ml.ruby-lang.org. I'll pick your suggestion.